### PR TITLE
[GH-243] Added udrl cut logic.

### PR DESCRIPTION
**Pull request by alanxoc3**
4 direction cutting on the grid now supported. Default keybindings are WASD (capitalized).

### CODE REVIEW

File: src/grid.c [hunk 1]
```diff
@@ -145,6 +149,38 @@ struct input_event *grid_mode()
 			redraw(mx, my, 0);
 		}
 
+		if (config_input_match(ev, "grid_cut_up")) {
+			my = my - grid_height / 2;
```
Comment (rvaiya) [inline src/grid.c:153]: This section should probably just be:
```
my -= grid_height / 4;
grid_height /= 2;
```

etc

**Review by rvaiya — COMMENTED** [review]
Looks good, just needs a few small tweaks.

**Comment by alanxoc3** [comment]
updated

**Comment by rvaiya** [comment]
Thanks, merged.

**Pull request merged**